interactive_templates/templates/v2/analysis/measures.py

```diff
@@ -160,6 +160,13 @@ def main():
                     "70-79",
                     "80+",
                 ],
+                "imd": [
+                    "Most deprived",
+                    "2",
+                    "3",
+                    "4",
+                    "Least deprived",
+                ],
```

Comment:
Just checking my understanding of the code- this means those with missing IMD will be excluded from ALL measures, correct? Or those not matching the above age bands would be excluded from ALL measures?
Would it be more flexible to retain missing here, and exclude missing from the plots by excluding them from the "category_order" list?

Reply:
Yeh you're right, for age and sex, this filter won't remove many people so I think it's ok to do it at this stage, but missing imd is more common (~1%), so make sense to do this later. I'll make that change